README.md

```diff
@@ -1,13 +1,18 @@
 # Happy Thoughts
+This project was to build a twitter-like app and using useState, useEffect and also both GET and POST from an API.
+We were also to copy a finished design as close as possible.
```

Comment:
Cute project that has the main characteristics from the design brief but with your own touch.